concepts/bit-manipulation/links.json

```
@@ -0,0 +1,14 @@
+[
+  {
+    "url": "https://crystal-lang.org/reference/1.10/syntax_and_semantics/operators.html#binary",
```

```suggestion
    "url": "https://crystal-lang.org/reference/syntax_and_semantics/operators.html#binary",
```

To not have language version-specific docs so does the version need to be dropped. Crystal gets a new release every 3 months, docs usually do not update that often but if it does it is likely better to just be on the latest version.